exceptional_operations.md

```diff
@@ -39,30 +39,30 @@ where attackers compromise only the Director repository.  However, there are two
 * A larger automaker might choose to deploy a dealer portal (i.e. a private, authenticated website) to allow authorized service centers to enter the details of the new ECU configuration themselves.
 
-Another option for updating the ECU configuration is to have a process that temporarily "unlocks" an ECU configuration, allowing the vehicle's Primary to directly report its new configuration (as opposed to having the mechanic enter the details of the replaced ECU). There is a tradeoff here: it streamlines the repair process, but more automation increases the risk that a real attack could go unnoticed.
+Another option for updating the ECU configuration is to have a process that temporarily "unlocks" an ECU configuration, allowing the vehicle's Primary to directly report its new configuration (as opposed to having the mechanic enter the details of the replaced ECU). There is a tradeoff here. While it streamlines the repair process, automating this step increases the risk that a real attack could go unnoticed.
 
 Note, however, that these are only recommendations. Uptane does not prescribe a protocol for this use case, because it is an orthogonal problem to software _update_ security. The advantage of this approach is that an OEM is free to solve this problem using existing solutions that it may already have in place.
 
 ### Aftermarket ECUs
 
-A slightly more difficult use case to deal with are aftermarket ECUs--for example, 3rd-party replacement parts, or add-on ECUs that add functionality for commercial fleet management. One approach is to work with the ECU manufacturer, and treat them like any other tier-1. (The addition of the aftermarket ECU would be managed in one of the ways recommended in the previous section.) However, this is likely not economically feasible in many/most cases. The easiest alternative is to simply exclude the aftermarket ECU from receiving OTA updates.
+A slightly more difficult use case to deal with are aftermarket ECUs--for example, 3rd-party replacement parts, or add-on ECUs that add functionality for commercial fleet management. One approach is to work with the ECU manufacturer and treat them like any other tier-1. (The addition of the aftermarket ECU would be managed in one of the ways recommended in the previous section.) However, this is likely not economically feasible in many/most cases. The easiest alternative is to simply exclude the aftermarket ECU from receiving OTA updates.
 
-Many aftermarket ECUs for fleet management, monitoring, or  have their own independent internet connection, and thus do not need to be integrated into the OEM's update system at all.
+Many aftermarket ECUs for fleet management or monitoring have their own independent internet connection, and thus do not need to be integrated into the OEM's update system at all.
```

Comment:
```suggestion
Some aftermarket ECUs, such as those designed for fleet management or monitoring, may have their own independent internet connection, and thus do not need to be integrated into the OEM's update system at all.
```
I think this flows a bit better. As far as the mysterious missing third item in the list, I think it's fine as it is. That bit was added by @tkfu, so if you remember whatever you might've been thinking of when you wrote that, feel free to add it back in!

Reply:
Whatever I was thinking of has been lost to the mists of time, but I agree that this flows well and gets the point across.